bot/exts/filters/filtering.py

```diff
@@ -413,7 +413,7 @@ async def _filter_message(self, msg: Message, delta: Optional[int] = None) -> No
                             await context.invoke(
                                 context.command,
                                 msg.author,
-                                arrow.utcnow() + AUTO_BAN_DURATION,
+                                datetime.utcnow() + AUTO_BAN_DURATION,
```

Comment:
```suggestion
                                arrow.utcnow().datetime + AUTO_BAN_DURATION,
```

Since the usage of `datetime.utcnow()` without a timezone is [not preferred](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow), and arrow is already imported, it might be better to still create an arrow time and pass the `datetime` property of it.

Reply:
fixed in https://github.com/python-discord/bot/pull/2250/commits/4eaecb32830aa122cd53656de562caf20b79e33d